src/interpreter/ByteCode.h

```diff
@@ -32,6 +33,8 @@ namespace Walrus {
 typedef uint16_t ByteCodeStackOffset;
 
 class FunctionType;
+class TagType;
+class Module;
```

Comment:
Are these needed?

Reply:
They were not actually needed, I have removed them.